modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Update `MatchFormat` to match multiple values on same element #TINY-7227
```

Comment:
This is more like:
Updated `formatter.matchFormat` to support for matching formats with variables in the classes property
or
Added support for matching formats with variables in the classes property

Reply:
updated